thorlcr/activities/thdiskbase.cpp

```diff
@@ -319,7 +318,6 @@ void CWriteMasterBase::publish()
                     partDesc->queryProperties().setPropInt64("@size", 0);
                     p++;
                 }
-                clusterIdx++;
```

Comment:
This looks like a bugfix rather than a pure warning removal, and should perhaps be in a separate jira?

Reply:
yes, I suspect it's [almost] benign, in that multi cluster target files on a remote target that's a different width to the local cluster (needs to be larger) are probably something that has never been used.
I'll remove and separate.